client/cmdlf.c

```diff
@@ -1044,6 +1045,7 @@ static command_t CommandTable[] =
 	{"awid",        CmdLFAWID,          1, "{ AWID RFIDs...    }"},
 	{"cotag",       CmdLFCOTAG,         1, "{ COTAG RFIDs...   }"},
 	{"em",          CmdLFEM4X,          1, "{ EM4X RFIDs...    }"},
+	{"fdx",         CmdLFFdx,           1, "{ FDX-B tags...    }"},
 	{"gproxii",     CmdLF_G_Prox_II,    1, "{ G Prox II tags...    }"},
```

Comment:
hehe, well,.. this is also a non-issue for me.  I just want to have a unified syntax and nomenclature. 
its kind of hard separating them, because of the different chipsets. I don't think the average lf users will understand the difference between your definition of tags, chipsset, ...